lib/registrant.rb

```diff
@@ -0,0 +1,17 @@
+class Registrant
+    attr_reader :name, 
+                :age, 
+                :permit,
```

Comment:
Careful here, we don't want a method called `.permit`, we wanted a method called `.permit?` since that will return a boolean, so `:permit` should not be in the attr_reader but we should have a method defined below called `.permit?`